infra-as-code/bicep/orchestration/hubPeeredSpoke/hubPeeredSpoke.bicep

```
@@ -0,0 +1,206 @@
+/*
+SUMMARY: This module acts as an orchestration module that create and configures a spoke network to deliver the Azure Landing Zone Hub & Spoke architecture
+DESCRIPTION:
+
+AUTHOR/S: KiZach
+VERSION: 1.0.0
+*/
+
+// **Parameters**
+// Generic Parameters - Used in multiple modules
+@description('The region to deploy all resoruces into. DEFAULTS TO = northeurope')
+param parLocation string = 'westeurope'
+
+@description('Prefix for the management group hierarchy. DEFAULTS TO = alz')
+@minLength(2)
+@maxLength(10)
+param parTopLevelManagementGroupPrefix string = 'alz'
+
+@description('Subscription Id to the Virtual Network Hub object. DEFAULTS TO empty')
+param parPeeredVnetSubscriptionId string = ''
+
+@description('Array of Tags to be applied to all resources in module. Default: empty array')
+param parTags object = {}
+
+@description('Set Parameter to true to Opt-out of deployment telemetry DEFAULTS TO = false')
+param parTelemetryOptOut bool = true
+
+// Subscription Module Parameters
+@description('The Management Group Id to place the subscription in. DEFAULTS TO empty')
+@allowed([
+  'landingZonesCorp'
+  'identity'
+  'connectivity'
+  ''
+])
+param parPeeredVnetSubscriptionMGPlacement string = ''
+
+// Resource Group Module Parameters
+@description('Name of Resource Group to be created to contain spoke networking resources like the virtual network.  Default: {parTopLevelManagementGroupPrefix}-{parLocation}-spoke-networking')
+param parResourceGroupNameForSpokeNetworking string = '${parTopLevelManagementGroupPrefix}-${parLocation}-spoke-networking'
+
+// Spoke Networking Module Parameters
+@description('Existing DDoS Protection plan to utilize. Default: Empty string')
+param parDdosProtectionPlanId string = ''
+
+@description('The Name of the Spoke Virtual Network. Default: vnet-spoke')
+param parSpokeNetworkName string = 'vnet-spoke'
+
+@description('CIDR for Spoke Network. Default: 10.11.0.0/16')
+param parSpokeNetworkAddressPrefix string = '10.11.0.0/16'
+
+@description('Array of DNS Server IP addresses for VNet. Default: Empty Array')
+param parDNSServerIPArray array = []
```

```suggestion
param parDnsServerIpArray array = []
```
we are adopting camelCasing throughout all our modules in an upcoming PR so this just gets us a little ahead 👍

---

Updated in newest commit